threads/views.py

```
@@ -127,9 +127,50 @@ class ReplyViewSet(viewsets.ModelViewSet):
     queryset = Reply.objects.all()
     serializer_class = ReplySerializer
 
+    def get_queryset(self, *args, **kwargs):
+        return Reply.objects.filter(comment=self.get_comment())
+
+    def get_comment(self):
+        try:
+            return Comment.objects.get(id=self.kwargs["comments_pk"])
+        except Comment.DoesNotExist:
+            raise PermissionDenied("Comment not found")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)